### PR TITLE
Add Sequence to onboarding docs

### PR DESCRIPTION
**Pull request by taylanpince**
**What changed? Why?**
Adding [Sequence](https://sequence.xyz/) to onboarding docs.

**Notes to reviewers**
N/A

**How has it been tested?**
Links have been tested, Sequence added in alphabetical order following convention.

**Does this PR add a new token to the bridge?**
No

Are you adding an entry to [`assets.ts`](../apps/bridge/assets.ts)?

- [x] No, this PR does not add a new token to the bridge
- [ ] Yes, and I've confirmed this token doesn't use a bridge override
- [ ] Yes, and I've confirmed this token is an OptimismMintableERC20

If you are adding a token to the bridge, please include evidence of both confirmations above for your reviewers.

### CODE REVIEW

File: apps/base-docs/docs/tools/onboarding.md [hunk 1]
```diff
@@ -54,6 +55,12 @@ You can [get started with Privy here](https://docs.privy.io/guide/quickstart), a
 
 ---
 
+## Sequence
+
+[Sequence](https://sequence.xyz/) is the leading all-in-one development platform for integrating web3 into games. Onboard, monetize, grow, and retain players with Sequence’s award-winning technology including: non-custodial Embedded Wallets, white labeled marketplaces and marketplace API, indexer, relayer, node gateway, Unity/Unreal/React Native/Mobile SDKs, transaction API, contract deployment, analytics, and more. [Learn more here](https://sequence.xyz/) and start creating on [Sequence Builder](https://sequence.build/) now.
```
Comment (wbnns) [inline apps/base-docs/docs/tools/onboarding.md:60]: Could you please drop the superlative (s/a leading/the leading)?

**Review by wbnns — CHANGES_REQUESTED** [review]
@taylanpince 

Thanks for the PR! Left one requested change. Also, do you all have a Base-specific landing page you can link to instead of the homepage?

File: apps/base-docs/docs/tools/onboarding.md (hunk 1 above)
Comment (taylanpince) [reply apps/base-docs/docs/tools/onboarding.md]: @wbnns Thank you for the quick review! Both changes are added now.

**Review by wbnns — CHANGES_REQUESTED** [review]
@taylanpince 

Thanks! The landing page looks good! Can you please change:

`the all-in-one` to `an all-in-one` (remove superlative)

**Comment by cb-heimdall** [comment]
Review Error for wbnns @ 2024-04-30 11:58:09 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Comment by taylanpince** [comment]
> @taylanpince
> 
> Thanks! The landing page looks good! Can you please change:
> 
> `the all-in-one` to `an all-in-one` (remove superlative)

Done ✅

**Review by zencephalon — APPROVED** [review]

**Comment by taylanpince** [comment]
@wbnns Do you need anything else from our end to finalize this?

**Review by wbnns — APPROVED** [review]
@taylanpince 

Thanks for the additional changes!

**Pull request merged**